models/lobby/server.go

```diff
@@ -0,0 +1,228 @@
+package lobby
+
+import (
+	"log"
+	"time"
+
+	"github.com/TeamPlayTF/PlayerStatsScraper/steamid"
+	"github.com/TeamPlayTF/TF2RconWrapper"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type Server struct {
+	Map  string // lobby map
+	Name string // server name
+	Rcon *TF2RconWrapper.TF2RconConnection
+
+	League League
+	Type   LobbyType // 9v9 6v6 4v4...
+
+	Address string // server ip:port
+	LobbyId bson.ObjectId
+
+	Players        []TF2RconWrapper.Player // current number of players in the server
+	AllowedPlayers []TF2RconWrapper.Player
```

Comment:
maybe use a hash for this? Since you're only checking for existence, `_, ok := AllowedPlayers[id]; ok` is faster than looping through an array and checking against every element